data/src/main/java/com/prac/data/source/api/GitHubApi.kt

```diff
@@ -12,4 +13,11 @@ internal interface GitHubApi {
         @Query("per_page") perPage: Int,
         @Query("page") page: Int
     ): List<RepoDto>
+
+
+    @GET("user/starred/{userName}/{repoName}")
+    suspend fun checkRepositoryIsStarred(
+        @Path("userName") userName: String,
+        @Path("repoName") repoName: String
+    ): Response<Unit>
```

Comment:
아래처럼 response 를 없애고, run catching 만 해두면  굳이 response code 를 비교하지 않아도 성공인지 아닌지를 판단할 수 있어요!

```suggestion
    suspend fun checkRepositoryIsStarred(
        @Path("userName") userName: String,
        @Path("repoName") repoName: String
    )
```

Reply:
```kotlin
internal class RepoStarApiDataSourceImpl @Inject constructor(
    private val gitHubApi: GitHubApi
): RepoStarApiDataSource {
    override suspend fun checkRepositoryIsStarred(repoName: String) : Boolean {
        return try {
            gitHubApi.checkRepositoryIsStarred("GongDoMin", repoName)           
            true
       } catch (e: Exception) {
            false
       }
    }
}
```